pkg/apis/storage/validation/validation.go

```diff
@@ -44,8 +44,8 @@ const (
 	maxAttachedVolumeMetadataSize = 256 * (1 << 10) // 256 kB
 	maxVolumeErrorMessageSize     = 1024
 
-	csiNodeIDMaxLength       = 128
-	csiNodeIDLongerMaxLength = 192
+	csiNodeIDMaxLength       = 192
+	csiNodeIDLongerMaxLength = 256
```

Comment:
convenient that this swap was sufficient

lgtm once the spec change is merged and released